services/webhook/slack_test.go

```diff
@@ -170,3 +171,21 @@ func TestSlackJSONPayload(t *testing.T) {
 	require.NoError(t, err)
 	assert.NotEmpty(t, json)
 }
+
+func TestIsValidSlackChannel(t *testing.T) {
+	tt := []struct {
+		channelName string
+		expected    bool
+	}{
+		{"gitea", true},
+		{"  ", false},
+		{"#", false},
+		{" #", false},
+		{"gitea   ", true},
+		{"  gitea", true},
+	}
+
+	for _, v := range tt {
+		assert.Equal(t, v.expected, IsValidSlackChannel(strings.TrimSpace(v.channelName)))
+	}
```

Comment:
I do not think the `TrimSpace` should be used here, it's not part of the target (to be tested) function.

```suggestion
	tt := []struct {
		channelName string
		expected    bool
	}{
		{"gitea", true},
		{"", false},
		{"#", false},
	}

	for _, v := range tt {
		assert.Equal(t, v.expected, IsValidSlackChannel(v.channelName))
	}
```

Reply:
```suggestion
	tt := []struct {
		channelName string
		expected    bool
	}{
		{"gitea", true},
		{"  ", false},
		{"#", false},
		{" #", false},
		{"gitea   ", false},
		{"  gitea", false},
	}

	for _, v := range tt {
		assert.Equal(t, v.expected, IsValidSlackChannel(v.channelName))
	}
```